test_helpers/main.go

```diff
@@ -188,7 +187,7 @@ func StartTarantool(startOpts StartOpts) (TarantoolInstance, error) {
 		// Create work_dir for a new instance.
 		// TO DO: replace with `os.MkdirTemp` when we drop support of
 		// Go 1.16 an older
```

Comment:
Please, remove the note (and try to find all the same notes in the code).

Reply:
Updated. Removed more comments like this, updated code and `Makefile` a little.